app/Http/Controllers/Api/Web/AlbumsController.php

```diff
@@ -83,6 +86,53 @@ public function getShow($id)
         ], 200);
     }
 
+    public function getCachedAlbum($id, $format)
```

Comment:
This is a very non-trivial controller action - how much of the logic in it could you shuffle over to the `Album` class?